devnet/cmd/update-dd-agent-confd/main.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package main
+
+import (
+	"fmt"
+	"log"
+
+	_ "embed"
+
+	"github.com/jessevdk/go-flags"
```

Comment:
Can't you use the stdlib's `flag` package for this?

Reply:
I prefer `go-flags` and just using a struct.  It has served me well for previous projects.